controls/libyaml_library_exists.rb

```diff
@@ -29,10 +29,11 @@
   plan_pkg_version = (plan_pkg_ident.match /^#{plan_origin}\/#{plan_name}\/(?<version>.*)\//)[:version]
   # create pkgconfig filename using major.minor portion of the $plan_pkg_version; in other words,
   # 0.1.7 becomes 0.1 and the filename is therefore "yaml-0.1.pc"
-  pkgconfig_full_path = File.join(plan_installation_directory.stdout.strip, 'lib', 'pkgconfig',  "yaml-#{plan_pkg_version.gsub(/\.\d+$/, "")}.pc")
+  #  "yaml-0.1.pc" is fixed name. So hardcoded the same
```

Comment:
I would remove al 3 lines of comments here. The third simply negates the first 2 and it will probably be clearer if we remove them all.

Reply:
Removed the comments